etcdconfigweb/config.go

```diff
@@ -81,28 +107,45 @@ func (ch ConfigHandler) handleRequest(ctx context.Context, query url.Values, hea
 
 		// insert new node
 		err := ch.etcdHandler.CreateNode(ctx, pubkey, func(info *ffbs.NodeInfo) {
-			const V4_BASE uint32 = 10 << 24
-			const V4_RANGE_SIZE uint8 = 10
-			const V6_BASE_HIGH uint64 = 0x20010bf70381 << 16
+			const CLIENT_V4_RANGE_SIZE uint = 22
+			const CLIENT_V6_RANGE_SIZE uint = 64
+
+			// IPv4 handling
+			v4RangeStr := os.Getenv("PARKER_V4_RANGE")
```

Comment:
Personally would tend to use command line options (maybe in combination with a configuration file) for all the moving parts. Mostly because of the builtin documentation of the options via `--help` (and showing the default option values).